core/client/ui/toolboxes/level-toolbox.js

```diff
@@ -39,6 +39,24 @@ Template.levelToolbox.events({
     const { x, y } = user.profile;
     updateLevel(level.name, { x, y }, level.hide);
   },
+  'change .js-voice-amplifier-select'(event) {
+    const featuresPermissions = getFeaturesPermissions();
+    const level = currentLevel(Meteor.user());
```

Comment:
Let's avoid these two lines on each handler by not passing the whole `featuresPermissions` object to `updateLevel`, but just the value that changed.